staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -87,6 +87,12 @@ func (m *Helper) List(namespace, apiVersion string, options *metav1.ListOptions)
 		NamespaceIfScoped(namespace, m.NamespaceScoped).
 		Resource(m.Resource).
 		VersionedParams(options, metav1.ParameterCodec)
+
+	if len(options.Continue) > 0 {
```

Comment:
Is there a reason to do this over `options.Continue != ""`?

Reply:
I was thinking on the first request, to leave the rate limiter in place, but on subsequent “continue” requests to remove it. I was hesitant to completely remove throttling since I don’t have the background for why client side throttling was put in place to start with.

You’re probably right though, it probably doesn’t make much difference, and maybe the throttling should just be removed for list requests altogether, even the first request without a continue token.

@apelisse do you think I should should remove the if statement and just never throttle on list requests (even on the first call)?

Reply:
I don't know enough about the reasons why we have throttling here in the first place. Would you be able to dig through `git blame` to figure out who might know about that? Or we can just ping @liggitt :-)